src/domain/kaivuilmoitus/Contacts.tsx

```diff
@@ -86,10 +149,15 @@ export default function Contacts() {
           />
           <TextInput
             name="applicationData.invoicingCustomer.registryKey"
-            label={t('form:yhteystiedot:labels:yTunnusTaiHetu')}
+            label={getInvoicingRegistryKeyLabel(t, selectedContactType)}
             required
             autoComplete="on"
             defaultValue={null}
+            helperText={
+              isRegistryKeyHiddenNotificationShown
```

Comment:
Displaying helper text could be determined straight from `registryKeyHidden` without the need for useState and useEffect for that.

Reply:
Thank you. Changed it.